src/community/resources/index.md.njk

```diff
@@ -0,0 +1,103 @@
+---
+title: Resources
+description: Useful resources and community projects
+section: Community
+layout: layout-pane.njk
+order: 1
+status: Experimental
+---
+
+A curated list of community developed resources that can help you use the GOV.UK Design System in your chosen design tool or development environment. 
+
+The Design System team is not responsible for these resources and is unable to provide support for them. If you have support or feature requests please contact the owner.
+
+
+## Create flow diagrams
+
+[GOV.UK Design System Flow Diagrams](https://github.com/dashouse/govuk-design-system-flow-diagrams)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow](https://github.com/charlesrt/gov-flow)⁠ -  
+A Sketch file for creating flow diagrams of GOV.UK services
+
+[GOV Flow for Google Drawings](https://github.com/charlesrt/gov-flow)⁠ -  
+Flow diagram templates for Google drawings, based on GOV Flow
+
+https://www.beatnic.co.uk/2019/10/04/google-drawing-template-gov-flow-kit/
+
+[GOV.UK Userflow Icons](https://github.com/alirawashdeh/govuk-userflow-icons) -  
+SVG images for creating flow diagrams of GOV.UK services
+
+## Create prototypes and wireframes
+
+[govuk-design-system-balsamiq](https://github.com/enoranidi/govuk-design-system-balsamiq)⁠ -  
+Balsamiq wireframes based on the GOV.UK Design System
+
+[Sketch Wireframing Kit](https://github.com/abbott567/sketch_wireframing_kit)⁠ -  
+Sketch wireframes based on the GOV.UK Design System
+
+## Build front ends
+
+### C#
+
+[GOV.UK Design System in C#/ASP.Net Core](https://github.com/cabinetoffice/govuk-design-system-dotnet)⁠ -  
+A collection of GOV.UK Frontend components built in C# / ASP.Net Core
+
+
+### Node.js
+
+[Hapi GOV.UK examples](https://github.com/DEFRA/hapi-govuk-examples)⁠ -  
+An introduction to building a hapi service using GOV.UK Frontend
+
+
+### PHP
+
+[Drupal theme](https://www.drupal.org/project/govuk_design_system)⁠ -  
+A Drupal theme that uses GOV.UK Frontend
+
+
+### Python
+
+[Python Microservice](https://github.com/willp-bl/python-microservice)⁠ -  
+A Flask/Python implementation of GOV.UK Frontend
+
+[GOV.UK Frontend Jinja ](https://github.com/alphagov/govuk-frontend-jinja)⁠ -  
+Classes and modules for using GOV.UK Frontend in your Jinja2-powered Python app
+
+[gov.js](https://gist.github.com/andymantell/8b1bb037341f38990a4d019894abd83e)⁠ -  
+A Gulp task to convert GOV.UK Frontend Nunjucks macros to Jinja2
+
+### R
+
+[Govdown](https://github.com/ukgovdatascience/govdown)⁠ -  
+A GOV.UK Design System theme for R Markdown
+
+### Ruby on Rails
+
+[GOV.UK Frontend Rails](https://github.com/dxw/dxw_govuk_frontend_rails)⁠ -  
+A Ruby Gem that adds the GOV.UK Frontend to Rails apps
+
+[GOV.UK Rails Boilerplate](https://github.com/DFE-Digital/govuk-rails-boilerplate)⁠ -  
+A complete Rails starter app using GOV.UK Frontend
```

Comment:
```suggestion
A complete Rails starter application using GOV.UK Frontend
```
Are we sure this is 'complete'?

Reply:
I added the word 'complete' in #1295. It's _complete_ in that the repo contains [a blank Rails app with all of the GOV.UK frontend assets configured and loaded](https://govuk-rails-boilerplate.herokuapp.com/pages/home). Almost all DfE Rails projects are started with it.